crates/fuel-core/src/service/sub_services.rs

```diff
@@ -280,6 +280,11 @@ pub fn init_sub_services(
 
     let graphql_block_importer =
         GraphQLBlockImporter::new(importer_adapter.clone(), import_result_provider);
+    let base_asset_id = config
+        .snapshot_reader
+        .chain_config()
+        .consensus_parameters
+        .base_asset_id();
```

Comment:
```suggestion
    let base_asset_id = *chain_config.consensus_parameters. base_asset_id();
```

Reply:
Updated in a915976cc879aa058264fe0fd4a24168eddd8cac